### PR TITLE
integrate HCI_VS_MSFT_LE_Monitor_Advertisement [v2] into Microsoft-defined Bluetooth HCI extensions

### PR DESCRIPTION
**Pull request by erikpe-msft**
This change:
* integrates all of the partner-reviewed changes to the Microsoft-defined Bluetooth HCI extensions specification to include the HCI_VS_MSFT_LE_Monitor_Advertisement [v2] command
* adds a (rewritten and updated) Visio version of the HCI_VS_MSFT_LE_Monitor_Advertisement state diagram and an exported .svg for the spec
* adds some new text around the diagram to integrate it a bit better with the original text

### CODE REVIEW

File: windows-driver-docs-pr/bluetooth/microsoft-defined-bluetooth-hci-commands-and-events.md [hunk 1]
```diff
@@ -267,23 +269,76 @@ HCI_VS_MSFT_LE_Monitor_Advertisement requests that the controller starts monitor
...
 
-:::image type="content" source="images/HCI_VS_MSFT_LE_Monitor_Advertisement_State_Diagram.png" alt-text="State diagram for HCI_VS_MSFT_LE_Monitor_Advertisement.":::
+:::image type="content" source="images/HCI_VS_MSFT_LE_Monitor_Advertisement_State_Diagram.svg" alt-text="State diagram for HCI_VS_MSFT_LE_Monitor_Advertisement.":::
+
+The controller shall begin monitoring an advertisement only when the received RSSI is greater than or equal to *RSSI_threshold_high* for a particular device and the *Monitor_options* match (see below).  The controller shall generate an [HCI_VS_MSFT_LE_Monitor_Device_Event][ref_HCI_VS_MSFT_LE_Monitor_Device_Event] with *Monitor_state* set to 1 and *Monitor_handle* set to the handle for this *Condition*, to notify the host that the controller is monitoring this particular device for *Condition*. Additionally, the Controller shall propagate the first advertisement report of a monitored advertisement to the Host only when the *Advertisement_report_filter_options* match (see below).
+
+The *Monitor_options* for a filter are considered a match based on the following logic:
+
+```
```
Comment (mhopkins-msft) [inline windows-driver-docs-pr/bluetooth/microsoft-defined-bluetooth-hci-commands-and-events.md:290]: Be sure to mark all the code blocks with the dev language so the syntax coloring is rendered properly. A C++code block looks like this:

\```cpp
 // C++ code goes here
\```

Comment (mhopkins-msft) [reply windows-driver-docs-pr/bluetooth/microsoft-defined-bluetooth-hci-commands-and-events.md:290]: Actually, this looks like pseudo-code. We don't really have a way to specify pseudo-code.

Comment (erikpe-msft) [reply windows-driver-docs-pr/bluetooth/microsoft-defined-bluetooth-hci-commands-and-events.md:290]: Thanks Mark! These are just pseudocode to help explain the logic. I wonder if there's a markdown code block language that would give them some basic highlighting...? I could try a few to see what happens.

Comment (erikpe-msft) [reply windows-driver-docs-pr/bluetooth/microsoft-defined-bluetooth-hci-commands-and-events.md:290]: Unfortunately, `cpp` behaves pretty strangely with the sorta-English fragments in there. I should definitely indicate that it isn't meant to be "real" code, but I'm at a loss for a lexer that nicely highlights the operators without tripping over everything else.

Comment (mhopkins-msft) [reply windows-driver-docs-pr/bluetooth/microsoft-defined-bluetooth-hci-commands-and-events.md:290]: I guess we just need to leave the dev language unspecified. The only other solution I can think is to not use a code block at all, just describe the logic in the text.

**Comment by mhopkins-msft** [comment]
Hi @erikpe-msft,

If you're good with it at this point, I'll go ahead and merge it.

Thanks for the PR!
 ~ Mark

**Comment by erikpe-msft** [comment]
Give me just a bit to see if anyone else on my team wants to give it a look...

---
In reply to: [1548290163](https://github.com/MicrosoftDocs/windows-driver-docs/pull/3489#issuecomment-1548290163) [](http://example.com/codeflow?ancestors=1548290163,1548290163)

File: windows-driver-docs-pr/bluetooth/microsoft-defined-bluetooth-hci-commands-and-events.md (hunk 1 above)
Comment (erikpe-msft) [reply windows-driver-docs-pr/bluetooth/microsoft-defined-bluetooth-hci-commands-and-events.md:290]: Yeah, I originally tried explaining the logic in text, but it got pretty hard to express all of the conditions..

**Comment by erikpe-msft** [comment]
okay @mhopkins-msft , let's go ahead and merge it. if there's any additional feedback, we can spin another quick PR. Thanks!

---
In reply to: [1548467120](https://github.com/MicrosoftDocs/windows-driver-docs/pull/3489#issuecomment-1548467120) [](http://example.com/codeflow?ancestors=1548467120,1548290163,1548290163)

**Comment by mhopkins-msft** [comment]
@erikpe-msft it's merged. You can delete your branch.

We should also talk about using the internal repo going forward, since you are MSFT.

**Pull request merged**